src/scenarios/mauiandroid/pre.py

```diff
@@ -11,9 +11,14 @@
 from test import EXENAME
 
 setup_loggers(True)
+logger = getLogger(__name__)
+logger.info("Starting pre-command for Maui Android Default App")
 
 precommands = PreCommands()
-install_versioned_maui(precommands)
+
+precommands.setup_workload_update_mode("manifests")
```

Comment:
Should this be the default behavior? 
My understanding is that we have to set the update mode only if we want to use the nightly builds

Reply:
> My understanding is that we have to set the update mode only if we want to use the nightly builds

This is true. 

> Should this be the default behavior?

The change is intended to be the new default, and I wasn't planning for use to have 2 separate pipelines, one of released and another of nightly builds. These measurements should serve as a tool for internal monitoring and not public benchmarks. Do you see benefits in having both scenarios (released/nightly) running in the lab?

Reply:
If we checkout the repo and want to run the scenario locally with a specific release how would we do that?

Reply:
I see, I have not considered this as a user scenario when designing this change. 

We can specify local dotnet install by `. ./init.sh -dotnetdir <custom dotnet directory>` so I think it could be enough to pass an argument to the `pre.py` command to skip installing the additional workloads and rely on the user that the `<custom dotnet directory>` contains the specific release installed. Would that be sufficient for the use case you have in mind?

Reply:
Update on that, we actually already support this scenario. If you use:
1. `. ./init.sh -dotnetdir <custom dotnet directory>`
2. `pre.py <other_args> --readonly-dotnet --has-workload`

This should ensure that the test app is build using the locally installed dotnet that has the specific configuration of SDKs.